packages/commonwealth/client/scripts/helpers/token_balance_helper.ts

```diff
@@ -0,0 +1,42 @@
+import BN from 'bn.js';
+import app from 'state';
+import {
+  ChainNetwork,
+  ContractType,
+} from '../../../../common-common/src/types';
+import axios from 'axios';
+
+export const getTokenBalance = async () => {
```

Comment:
Should have some inline docs explaining what this function does, because it's not straightforward...

Reply:
In progress